### ISSUE TITLE
why the loss is always so small

### ISSUE THREAD

**Issue by vgg4resnet** [CVxTz/fingerprint_denoising#2]

**Comment by vgg4resnet** [comment]
I am trying to train my own data but the model‘s loss is very small，it is strange that the loss is that small at the begining of train，and the final trained model do not have any effect at all 
where do I make it wrong ，is this cause by the  loss fuction，should I try other fuction ？，I change the input to 96*96,  any suggestion，thanks in advance！ 



Epoch 00016: val_loss did not improve from 0.07810
Epoch 17/1000
 - 6s - loss: 0.1285 - mean_absolute_error: 0.2646 - val_loss: 0.0873 - val_mean_absolute_error: 0.2896

Epoch 00017: val_loss did not improve from 0.07810
Epoch 18/1000
 - 6s - loss: 0.1284 - mean_absolute_error: 0.2647 - val_loss: 0.0807 - val_mean_absolute_error: 0.2787

Epoch 00018: val_loss did not improve from 0.07810
Epoch 19/1000
 - 6s - loss: 0.1284 - mean_absolute_error: 0.2645 - val_loss: 0.0833 - val_mean_absolute_error: 0.2830

Epoch 00019: val_loss did not improve from 0.07810
Epoch 20/1000
 - 6s - loss: 0.1285 - mean_absolute_error: 0.2644 - val_loss: 0.0849 - val_mean_absolute_error: 0.2854

Epoch 00020: val_loss did not improve from 0.07810
Epoch 21/1000
 - 6s - loss: 0.1286 - mean_absolute_error: 0.2650 - val_loss: 0.0810 - val_mean_absolute_error: 0.2787

Epoch 00021: val_loss did not improve from 0.07810
Epoch 22/1000

**Comment by vgg4resnet** [comment]
another question，
my train model‘s output is all the  same，the predicted image values are all one number，
that makes np.max(pred_)-np.min(pred_)=0，cause
  pred_ = 255.*(pred_ - np.min(pred_))/(np.max(pred_)-np.min(pred_)) exception
but  I use your model it is normal，it can predict a right image，help appreciate

**Comment by CVxTz** [comment]
Hello,

You have some convergence issue in your network. Maybe the data is badly
shaped or the learning rate is too high or too low.
I can't really help you more without a running example. You can maybe setup
a google colab notebook with the data and I'll have a look at it.

Best,


Le jeu. 29 août 2019 à 03:43, vgg4resnet <notifications@github.com> a
écrit :
